src/KeyDB.php

```diff
@@ -55,18 +55,6 @@ public static function createFromRootMetadata(RootMetadata $rootMetadata, bool $
         return $db;
     }
 
-    /**
-     * Gets the supported encryption key types.
-     *
-     * @return string[]
-     *     An array of supported encryption key type names (e.g. 'ed25519').
-     */
-    public static function getSupportedKeyTypes(): array
```

Comment:
Was there any reason this was public before? If are going to put into validation should we just add note to the readme saying we just support this 1 type?

Reply:
After discussion, I agreed to mention this in in the wiki, since it's not very discoverable otherwise. https://github.com/php-tuf/php-tuf/wiki#notes